packages/node_modules/@ractivejs/cli/src/index.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-env node */
+/* eslint-disable no-console */
+import serialize from 'serialize-javascript'
+import program from 'commander'
```

Comment:
Does this mean that the cli gets a dependency on commander? It seems to be missing from the package?

Reply:
The cli is built just like the other packages. It builds to `bin` and bundles commander, which conveniently has no dependencies other than native modules. So once published, the cli's only dependency is a peer dependency on core.